test/unittest/tests/test_fnv.cpp

```diff
@@ -0,0 +1,37 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include <catch2/catch.hpp>
+
+#include "wt_internal.h"
+
+// due credit: the cases that hash to zero came from
```

Comment:
Missing a capital letter at the start of the sentence. I've tagged a few more occurrences below with "Caps"

Reply:
Fixed.